packages/ckeditor5-package-generator/lib/index.js

```diff
@@ -64,7 +64,7 @@ new Command( packageJson.name )
  * @param {String} packageName
  * @param {CKeditor5PackageGeneratorOptions} options
  */
-async function init( packageName, options ) {
+async function init( packageName = '', options ) {
```

Comment:
The order of these arguments is not valid. Optional arguments should always be **after** the required ones.

Hence, I suggest a change:

* The `init()` function accepts a single object as an argument.
* `packageName` should be a part of the `options` object.

This way, you can call the `init()` function without specifying the package name. I know it is not valid from the script point of view (as the name is obligatory to generate anything), but it makes the design clear from the code point of view.

Reply:
Oh no :astonished: How I missed it?!

@przemyslaw-zan, could you please handle it?

Reply:
I extracted it to a new issue (https://github.com/ckeditor/ckeditor5-package-generator/issues/72), so we don't have to update anything here now.